frontend/src/pages/Sales/Sales.jsx

```diff
@@ -21,13 +21,24 @@ export default function Sales() {
     fetchSales();
   }, []);
 
+  let currentCategory = "";
   return (
     <div className="w-1/3 flex flex-col justify-center gap-8">
       <Banner bannerText="In de aanbieding" />
       <ul className=" flex flex-col gap-4">
-        {sales.map((product) => (
-          <ProductBox key={product.name} product={product} />
-        ))}
+        {sales.map((product) => {
+          if (product.category === currentCategory) {
+            return <ProductBox key={product.name} product={product} />;
+          } else {
+            currentCategory = product.category;
+            return (
+              <>
```

Comment:
this <> needs a key, instead of the productbox below

Reply:
updated with key and correct styling